chcemvediet/templates/main/homepage/about.html

```diff
@@ -14,7 +14,7 @@
   <div class="col-md-4 col-sm-6 chv-justify">
```

Comment:
V reze okolo 1100px je posunuty button:

![1100](https://user-images.githubusercontent.com/5547275/119355118-2ddcc180-bca5-11eb-9365-631ac0ae664c.png)

Reply:
@viliambalaz Ak sa ti to nepodari vyriesit, tak PR mergnime a toto nechajme na samostatny bug. Nech nas nezdrzuje. Lebo opravu layoutu ako taku by sa zislo deploynut. 

V tom pripade potom pls skopiruj tento koment (a obrazok) a sprav z neho novy bug issue.